### DISCUSSION TITLE
Using MX Master 3 for Mac gesture button and mapping to moving the pointer

### DISCUSSION THREAD

**Discussion by FMJansen** [linearmouse/linearmouse#766] [Q&A] [answered]
**TL;DR: can I map an action to both a specific button and the direction of pointer movement?**

Hey! I’m trying to switch away from Logi Options+, so I’m trying to configure linearmouse the way I want it. One part is using the Gesture button and dragging the mouse (moving the pointer) to switch spaces. I’ve tried and found some things, but I’m not there yet. 

What I’ve done:

- Install linearmouse
- Install Karabiner Elements (because @lujjjh mentioned it [under](https://github.com/linearmouse/linearmouse/issues/504#issuecomment-1595792461) #504 and it’s explained at a [hammerspoon issue](https://github.com/Hammerspoon/hammerspoon/issues/3373#issuecomment-1416548318) and a [post](https://scribe.rip/macbook-m1-and-mx-master-3-setting-buttons-without-logioptions-337790ba4cb3) linked from there
- I then mapped the gesture button with Karabiner Elements `left_command` to `button6` and `tab` to `vk_none` for just the MX Master 3
- In linearmouse I’ve configured the mouse through Karabiner (had already set up the MX Master 3, but since it’s routed/mapped through Karabiner, it becomes a different device for linearmouse) with e.g. the reverse scrolling direction I wanted
- Then I tried mapping `Button #5` (which is `button6` in Karabiner I believe) to `"missionControl.spaceRight"`
- However, I can’t seem to find in the [Configuration.md](https://github.com/linearmouse/linearmouse/blob/main/Documentation/Configuration.md#button-mappings) how to map not just buttons but also pointer movement to a specific action; so I can only move right, not move left or right based on which direction I move my mouse/pointer

Is there a way to also use pointer movement/direction in a mapping? Am I on the right track?

**Comment by lujjjh** [comment, marked as answer]
Unfortunately, LinearMouse currently does not support mapping dragging gestures.

**Reply by FMJansen** [reply]
Ah that’s a pity, thanks though!
